RecoTracker/LSTCore/src/alpaka/LSTEvent.dev.cc

```diff
@@ -324,6 +347,16 @@ void LSTEvent::createSegmentsWithModuleMap() {
 
 void LSTEvent::createTriplets() {
   if (!tripletsDC_) {
+    auto const countSegConn_wd = cms::alpakatools::make_workdiv<Acc3D>({nLowerModules_, 1, 1}, {1, 8, 32});
```

Comment:
I see that for segments both the counting and the creation grid sizes were made in sync.
Here this was not done.
`createTriplets_workDiv` block size is still `{1, 16, 16}`.
Was the counting grid size tuned in this PR or was it copy-pasted?

Reply:
Thanks for catching this, will fix.